keps/sig-api-machinery/4222-cbor-serializer/README.md

```diff
@@ -824,11 +861,16 @@ in back-to-back releases.
 - Client generation updated to support CBOR behind client-side gates.
 - Runtime gating mechanism added to client-go.
 - Maintenance of CBOR library is understood.
+- Types that implement json.Marshaler or json.Unmarshaler without corresponding custom CBOR
+  behaviors are either rejected with an error on Encode and Decode or automatically transcoded from
+  JSON.
 
 #### Beta
 
 - Review of nondeterministic encoding mode and final decision on whether to keep
   or remove it.
+- At least one alpha release has supported automatic transcoding of types that implement
+  json.Marshaler or json.Unmarshaler without corresponding custom CBOR behaviors.
```

Comment:
I believe this specified so we can be confident in a rolling upgrade.  If we relax that, this could be reconsidered.

Reply:
Correct, I edited the text to make the motivation explicit.